app/src/main/res/layout/activity_main.xml

```diff
@@ -5,6 +5,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/colorAccent"

```

Comment:
farba pozadia sa so zmenou temy nemeni, pretoze je tu definovana konkretna farba z palety farieb v colors.xml. aby sa aplikovala ta ktora je definovana pre vybranu temu, je potrebne aplikovat to co je ulozene pod colorAccent atributom. (?colorAccent)